portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsInterface.testcase

```diff
@@ -299,7 +299,7 @@ definition {
 
 		FormFields.viewFieldSize(
 			fieldName = ${fieldTextField1},
-			size = 6);
+			size = 5);
```

Comment:
Some fixes were made to form field panel resolutions which caused these tests to fail.

 As a solution, I put the correct sizes of the fields.